applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_flux_FIC_condition.cpp

```diff
@@ -201,13 +201,14 @@ template <unsigned int TDim, unsigned int TNumNodes>
 void UPwNormalFluxFICCondition<TDim, TNumNodes>::CalculateAndAddBoundaryMassMatrix(
     MatrixType& rLeftHandSideMatrix, NormalFluxVariables& rVariables, NormalFluxFICVariables& rFICVariables)
```

Comment:
I guess that due to your changes, it is now possible to use reference-to-const instead of just reference:
```suggestion
    MatrixType& rLeftHandSideMatrix, const NormalFluxVariables& rVariables, const NormalFluxFICVariables& rFICVariables)
```

Reply:
Added `const`.